owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -356,6 +358,9 @@ interface FileDao {
     @Query(UPDATE_FILE_WITH_NEW_AVAILABLE_OFFLINE_STATUS)
     fun updateFileWithAvailableOfflineStatus(id: Long, availableOfflineStatus: Int)
 
+    @Query(UPDATE_FILE_WITH_LAST_USAGE)
+    fun updateFileWithLastUsage(id: Long?, lastUsage: Long?)
```

Comment:
Same here as before, the ID shouldn't be nullable

Reply:
Done